zng/docs/spec.md

```diff
@@ -1,12 +1,15 @@
 # ZNG Specification
 
-> Note: This specification is ALPHA and a work in progress.
-> Zq's implementation of ZNG is tracking this spec and as it changes,
+> ### Note: This specification is ALPHA and a work in progress.
+> [Zq](https://github.com/brimsec/zq/blob/master/README.md)'s
+> implementation of ZNG is tracking this spec and as it changes,
 > the zq output format is subject to change.  In this branch,
 > zq attempts to implement everything herein excepting:
 >
 > * the `bytes` type is not yet implemented,
-> * the `enum` type is not yet implemented.
+> * the `enum` type is not yet implemented,
+> * only streams of `record` types (which may consist of any combination of
+>   other implemented types) may currently be expressed in value messages.
```

Comment:
I'm interested in feedback on if I'm capturing this correctly. I've confirmed via conversations & black box testing that `zq` doesn't support several of the examples here that are _not_ streams of records, e.g. a super simple example:

```
$ cat not-record.tzng 
#0:string
0:hello, world;

$ zq -t -i tzng not-record.tzng 
not-record.tzng: line 1: zng typedef not a record while parsing: "string"
```

Does this additional disclaimer covers the territory?

Reply:
Yes I think it does!